doc/api_2_0_spec/api_2_0_spec.md

```diff
@@ -771,6 +771,33 @@ the returned data would look like:
 }
```

Comment:
No table of contents entry. After adding a new API call (i.e. a new Markdown header), `cd doc/api_2_0_spec/` and run `./update_toc.sh MARKDOWN_FILE_YOU_HAVE_JUST_EDITED.md` which will autosupermagically update the Markdown ToC.

Reply:
fixed